settings.py

```diff
@@ -59,6 +59,7 @@ def env(key, default, cast=str):
     "tada",
 ]))
 
+SLACK_CUSTOM_EMOJI_MAPPING = env('SLACK_CUSTOM_EMOJI_MAPPING', json.dumps({}))
```

Comment:
You should fill this out with an example, so that other users know what to expect.

```python
json.dumps({
    "author": "custom_emoji"
})
```